dbms/src/Columns/ColumnFixedString.h

```diff
@@ -98,12 +98,20 @@ class ColumnFixedString final : public COWPtrHelper<IColumn, ColumnFixedString>
 
     void insertFrom(const IColumn & src_, size_t index) override;
 
+    void insertManyFrom(const IColumn & src_, size_t position, size_t length) override;
+
+    void insertDisjunctFrom(const IColumn & src_, const std::vector<size_t> & position_vec) override;
+
     void insertData(const char * pos, size_t length) override;
 
     void insertDefault() override
     {
         chars.resize_fill(chars.size() + n);
     }
+    void insertManyDefaults(size_t length) override
+    {
+        chars.resize_fill(chars.size() + n * length);
+    }
```

Comment:
Yeah, since we seldom used ColumnFixedString, just leave the code like this.

> `resize_fill` is equivalent to `resize` + `std::fill`, maybe `resize` + `memset` can get better performance?